### PR TITLE
cloud event docs #962

### PR DESCRIPTION
**Pull request by oraNod**
Docs for https://github.com/infinispan/infinispan-operator/pull/964

### CODE REVIEW

**Review by rigazilla — APPROVED** [review]
lgtm

File: documentation/asciidoc/topics/yaml/cloud_events.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+spec:
+  cloudEvents:
+    # Configures the number of acks.
+    acks: "1"
```
Comment (dmvolod) [inline documentation/asciidoc/topics/yaml/cloud_events.yaml:4]: just for information, acks could be number or "all" value, thanks the reason why it's a string
https://betterprogramming.pub/kafka-acks-explained-c0515b3b707e

Comment (oraNod) [reply documentation/asciidoc/topics/yaml/cloud_events.yaml:4]: nice one @dmvolod even though that might be kafka specific I've updated the comment with some additional detail

**Review by dmvolod — APPROVED** [review]

File: documentation/asciidoc/topics/yaml/cloud_events.yaml (hunk 1 above)
Comment (dmvolod) [reply documentation/asciidoc/topics/yaml/cloud_events.yaml:4]: @rigazilla could we add a validation for the acks values in CRD?

**Pull request merged**